tools/test_modules/m20700.pm

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env perl
+
+##
+## Author......: See docs/credits.txt
+## License.....: MIT
+##
+
+use strict;
+use warnings;
+
+use Digest::SHA  qw (sha256_hex);
+
+sub module_constraints { [[1, 31], [16, 16], [-1, -1], [-1, -1], [-1, -1]] }
```

Comment:
I think constraints for generic mode should be more open. Also note that for this mode you have pure and optimized kernels, therefore you should make use of all paramters not just for pure kernels.

Reply:
like this?

`sub module_constraints { [[1, 31], [16, 16], [1, 31], [16, 16], [-1, -1]] }`

Reply:
another thing, I need create two test modules right?
I need to rename this m20700.pm to m20711.pm and create the m20710 ?